.travis.yml

```diff
@@ -31,7 +32,16 @@ script:
   - cd acceptance-tests
   - bundle install
   - bundle exec spinach
-#  - kill %1
+  - kill %1
   - cd ..
 after_success:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then bash <(curl -s https://codecov.io/bash); fi
+#deploy:
```

Comment:
It'd be better not to include this if it's all commented out

Reply:
Good spot, thanks.